test/system/280-update.bats

```diff
@@ -349,4 +349,54 @@ function nrand() {
 
     run_podman rm -t 0 -f "$cid"
 }
+
+# bats test_tags=ci:parallel
+@test "podman update - set ulimits" {
```

Comment:
This is more or less the exact same test as e2e which means you are duplicated the test for no good reason. To keep CI speed as minimal as possible we should only have a one test. I think keeping the system test makes the most sense here.